recipe/meta.yaml

```
@@ -44,12 +66,18 @@ requirements:
     - scikit-build >=0.11.0
     - setuptools >=40.1.0
     - spdlog
+    - cuda-version {{ cuda_compiler_version }}  # [linux64 and cuda_compiler_version != "None"]
+    # Qiskit Aer now uses both
+    - custatevec                                # [linux64 and cuda_compiler_version != "None"]
+    - cutensornet                               # [linux64 and cuda_compiler_version != "None"]
```

Think we can do this

```suggestion
    - cuda-version {{ cuda_compiler_version }}  # [cuda_compiler_version != "None"]
    # Qiskit Aer now uses both
    - custatevec                                # [cuda_compiler_version != "None"]
    - cutensornet                               # [cuda_compiler_version != "None"]
```

---

Addressed in #21.